src/utils/constants.ts

```diff
@@ -53,7 +53,7 @@ export module REGEX {
     export const ROLE = /<@&(\d+)>/g;
     export const CHANNEL = /<#(\d+)>/g;
     export const USER = /<@(\d+)>/g;
-    export const URL = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/;
+    export const URL = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/g;
```

Comment:
You mean the g flag? g flag makes it do a global search - it'll find all occurrences. Previously, it would only have picked up the first URL; so if a message had multiple URLs it would only catch the first one.

Reply:
Yup that's what I meant 🙃